api-gateway/src/main/resources/static/scripts/visits/visits.controller.js

```diff
@@ -183,6 +185,50 @@ angular.module('visits')
             return practitionerName
         }
 
+        //Get Owners
+        var ownersUrl = "api/gateway/owners";
+        $http.get(ownersUrl).then(function (resp) {
+            self.owners = resp.data;
+            console.log(self.owners)
+        });
+
+        //get pets of owners
+        //whenever the owner is selected, this calls to the back end to get the pets of that owner
+/*        self.loadOwnerInfo = function() {
+            let ownerId = self.ownerId;
+            let petId = self.petId
+            $http.get("api/gateway/owners/" + ownerId).then(function(resp) {
+                self.pets = resp.data.pets;
+            });
+            console.log(ownerId)
+        };
+
+        self.logPetId = function() {
+            console.log("Selected Pet ID:", self.petId);
+        }*/
+
+        self.loadOwnerInfo = function() {
+            $http.get("api/gateway/owners/" + self.ownerId).then(function(resp) {
+                self.pets = resp.data.pets;
+                self.ownerId = $("#selectedOwner").val()
+                console.log(" Selected Owner ID:", self.ownerId);
+            });
+            let ownerIdI = self.ownerId
+            console.log(self.ownerId);
+        };
+
+        self.logPetId = function() {
+            let petIdI = self.petId
```

Comment:
I see that this petIdI constant is being set and the value of the classe's petId is being passed to it but I don't see this petIdI being used anywhere. I was just wondering what is it's purpose?

Reply:
Ya this was mainly for debugging and trying to store the pet id into the petIdI now the petIdI variable serves no purpose anymore since it was used for log statements which i removed